.dockerignore

```diff
@@ -12,9 +12,9 @@
 _kubevirtci
 
 # No need for source code is already compiled
-pkg
-cmd
-tools
+#pkg
```

Comment:
leftovers?

Reply:
No, this change is necessary because we need to build the two binaries listed [here](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/Makefile#L107-L111) inside the Dockerfile in order to support Go cross-compilation for building the target binaries.

Reply:
lets remove the lines then completely ?

Reply:
Yes, Removed in the latest commit.